book/src/snapshot-validation-webhook.md

```diff
@@ -0,0 +1,27 @@
+# Snapshot Validation Webhook
+
+## Status and Releases
+
+**Git Repository:** [https://github.com/kubernetes-csi/external-snapshotter](https://github.com/kubernetes-csi/external-snapshotter)
+
+**Status:** v3.0.1
+
+### Snapshot Validation Webhook
+
+There is a new validating webhook server which provides tightened validation on snapshot objects. This SHOULD be installed by all users of the snapshot feature.
```

Comment:
clarify here it should be the responsibility of k8s distros

Reply:
done